newIDE/app/src/ResourcesList/LocalPiskelBridge.js

```diff
@@ -47,17 +52,31 @@ export const openPiskel = ({
   ipcRenderer.removeAllListeners('piskel-changes-saved');
   ipcRenderer.on(
     'piskel-changes-saved',
-    (event, outputResources, newAnimationName) => {
+    (event, outputResources, newAnimationName, metadata) => {
+      const externalEditorData = metadata.data
```

Comment:
I'm still not sure why we are testing for `metadata.data` existence? What could be metadata?

Reply:
removing this causes gdevelop to create an empty pskl object. When the object is sent  to piskel, piskel thinks that it has layers and tries to load it with the wrong method. I need to change this